packages/controllers/tests/controllers/ConnectionController.test.ts

```diff
@@ -247,6 +247,11 @@ describe('ConnectionController', () => {
       polkadot: 'polkadot-connector',
       bip122: CommonConstantsUtil.CONNECTOR_ID.WALLET_CONNECT
     }
+    ChainController.state.chains.set('eip155', {
```

Comment:
Maybe use again `ConstantsUtil.CHAIN.EVM` as you did on `ChainController.test.ts`?

Reply:
Same for the belows